isamples_metadata/Transformer.py

```diff
@@ -24,6 +24,10 @@ class Transformer(ABC):
 
     N2T_ARK_NO_HTTPS_PREFIX = f"{N2T_NO_HTTPS_PREFIX}ark:/"
 
+    RULE_BASED_CONFIDENCE = 1.0
```

Comment:
New confidence values as discussed on Wednesday -- 0…1 is machine-based, 1.0 is rule-based, 2.0 is human-entered.